functions/main.py

```diff
@@ -0,0 +1,97 @@
+import json
+import requests
+from bs4 import BeautifulSoup
+
+def hello(data):
+    backtext = 'Hello world!\ndata:' + str(data) + '\n'
+    print(data)
+    # print(context)
+    return backtext
+
+def get_list_from_api(title):
+    print("title:" + title)
+    headers = {
+        "Connection": "keep-alive",
+        "accept": "*/*",
+        "x-app-id": "RUMORS_SITE",
+        "DNT": "1",
+        "User-Agent": "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/81.0.4044.62 Safari/537.36",
+        "content-type": "application/json",
+        "Origin": "https://cofacts.g0v.tw",
+        "Sec-Fetch-Site": "same-site",
+        "Sec-Fetch-Mode": "cors",
+        "Sec-Fetch-Dest": "empty",
+        "Referer": "https://cofacts.g0v.tw/",
+        "Accept-Language": "zh-TW,zh;q=0.9,en;q=0.8"
+    }
+    data_data  = { 
+        'operationName': 'ListReplies', 
+        'variables': { 
+            'filter': { 
+                'moreLikeThis': { 
+                    'like': 'N/A', 
+                    'minimumShouldMatch': '0' 
+                    } 
+                }, 
+        'orderBy': [{ 
+            '_score': 'DESC' 
+            },{ 
+            'createdAt': 'DESC' 
+            }]
+        }, 
+        'query': 'query ListReplies($filter: ListReplyFilter, $orderBy: [ListReplyOrderBy], $before: String, $after: String) { ListReplies(filter: $filter, orderBy: $orderBy, before: $before, after: $after, first: 10) { edges { node {  ...ReplyItem  __typename } cursor __typename } __typename } }  fragment ReplyItem on Reply { id text type createdAt user { id name __typename } articleReplies(status: NORMAL) { articleId replyId __typename } __typename } ' 
+    }
+    data_data['variables']['filter']['moreLikeThis']['like'] = title
+    data = json.dumps(data_data)
+    # data = json.dumps({"query":data_data})
+    # data = json.load(data)
+    # print(str(data))
+    try:
+        r = requests.post("https://cofacts-api.g0v.tw/graphql", headers=headers, data=data)
+        r.encoding = 'utf8'
+        print(str(r))
+        # print(str(r.text))
+        soup = BeautifulSoup(r.text, 'html.parser')
+        # print(soup)
+    except:
+        print('\033[33m Error! \033[0m')
+        return 'no'
```

Comment:
don't forget status code, and please consider message format, please use the same format otherwise it will be way too dynamics